GCP/legos/Save_CSV_to_Google_Sheets_v1/Save_CSV_to_Google_Sheets_v1.py

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
Could we follow the same Lego naming convetion. 

`connector`_`lego_name`.[.py,json]

In that aspect, the name would be 

`gcp_save_csv_to_google_sheets.py` and `gcp_save_csv_to_google_sheets.json`

Reply:
Also the folder can be `gcp_save_to_google_sheets` and it should also have a `__init__.py` file.
i have a bug to create __init__.py files automatically. I will get to it Next.

Reply:
fixed all of the above